flink-formats/flink-avro/pom.xml

```diff
@@ -64,8 +64,7 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<!-- use a dedicated Scala version to not depend on it -->
```

Comment:
I think this is dangerous, because it means that the artifact we release have an undefined Scala version - effectively whatever version we uploaded first (or last?).

That means `flink-avro` 1.7 depends on Scala 2.11, `flink-avro` 1.8 might depend on Scala 2.12,  `flink-avro` 1.9 might depend again on Scala 2.11.